restapi/src/test/java/io/stargate/web/docsapi/service/query/filter/operation/impl/GtFilterOperationTest.java

```diff
@@ -40,17 +40,17 @@ public void stringEquals() {
     }
 
     @Test
-    public void stringGreater() {
+    public void stringLess() {
       boolean result = gt.test("filterValue", "aaa");
 
-      assertThat(result).isTrue();
+      assertThat(result).isFalse();
```

Comment:
nit: WDYT about  `assertThatgt.test("filterValue", "aaa")).isFalse()`... I find it a bit hard to read through the `result` variable indirection.

Reply:
I think now it's OK to read after the change of params and the method name updates..